docs/local-setup.md

```diff
@@ -0,0 +1,52 @@
+## Backend 
+
+#### Start App
+1. Run `npm run init` first time to install dependencies and initialize your local database
+3. Run `npm run develop`
+4. Go to http://localhost:1337/admin and login using email `local-admin@devlaunchers.org` and password is `W&x5ZzOMtBCVt1YAUiuSzt~5`.
+
+#### Init command will do the following
+
+1. Runs NPM Install
+
+2. Builds Strapi Admin UI
+
+3. Set up default environment file into `.env`
+
+4. Starts up Strapi to allow database initialized
+
+5. Enables Google provider using test app client id and secret.
+
+6. Setup Authenticated and Public role permissions.
+
+7. Creates new Strapi admin account.
+
+8. Creates default interests and projects
+
+
+## FrontEnd
+
+#### Start App
+1. Update `env.development` to point to local backend
+
+```properties
+NEXT_PUBLIC_API_BASE_URL=http://localhost:1337
+NEXT_PUBLIC_API_URL=http://localhost:1337/api
+NEXT_PUBLIC_STRAPI_URL=http://localhost:1337/api
+NEXT_PUBLIC_GOOGLE_AUTH_URL=http://localhost:1337/api/connect/google
+```
+
+2. Install dependencies: `yarn install`
+
+3. Run the tailwindcss compiler: `yarn workspace @devlaunchers/tailwind dev`
+
+4. Run the app's development server: `yarn workspace @devlaunchers/app dev`
+
+4. Go to http://localhost:3000 and sign up
+
+5. After login, you should be able reproduce the steps from UI perspective
+
+# Cookies Issue
+
+After signing in DevLaunchers front end, login into Strapi Admin doesn't work due to some shared Cookies.
+It's suggested to open them in different browsers or use an incognito mode.
```

Comment:
is it possible to not have those cookies be shared? as far as I know that's not possible

Reply:
@Enjoy2Live 
Actually, it is possible to not have this issue if we agree to always refer to the FrontEnd as `http://127.0.0.1:3000` and to Backend as `http://127.0.0.2:1337`. That would be considered different domains. But we will need to update all references to `localhost`. Maybe that is something to discuss for monorepo integrations steps?

Reply:
So it's not possible to send cookies through a different port only, In this case only 3000? If that's the case then yes let's discuss it when migrating to monorepo

Reply:
Correct. Cookies are domain-only specific. Same domain with multiple different ports will end up using all cookies for a domain. Another thought would be to see which cookie is that and maybe there's a setting to name it differently in Strapi. I will have a look.

Reply:
@Enjoy2Live @chungthuang 
Turns out it is not a cookie issue but the session token we set over from cookie to a header. Which is great for /api endpoints but not so well for /admin. https://github.com/dev-launchers/strapiv4/pull/153/files#diff-3075c669577ff744720d664128b19bf879a1fe1e48cc878639f0ac7f24253d5fR8
There's a dedicated extension for this specific issue, from which I borrowed the check to apply Authorization header only to /api endpoints https://github.com/bwyx/strapi-jwt-cookies/blob/main/middlewares/getter.js#L8

Reply:
Great finding!